tools/bwa.wdl

```diff
@@ -166,13 +166,16 @@ task bwa_aln_pe {
         tar -C bwa_db -xzf ~{bwa_db_tar_gz} --no-same-owner
         PREFIX=$(basename bwa_db/*.ann ".ann")
 
+        ln -sf ~{read_one_fastq_gz}
+        ln -sf ~{read_two_fastq_gz}
```

Comment:
```suggestion
        ln -s ~{read_one_fastq_gz}
        ln -s ~{read_two_fastq_gz}
```
This will error on the case that a user gives a read1 and a read2 FASTQ with the same name. I'm not exactly sure what will happen with `-f` but this should be a clearer error message than relying on `bwa`